.github/workflows/build.yml

```diff
@@ -119,3 +123,4 @@ jobs:
           payload: "{\"type\":\"Build\",\"url\":\"${{ github.server_url }}/${{ github.repository }}/actions/runs/${{ github.run_id }}\"}"
         env:
           SLACK_WEBHOOK_URL: ${{ secrets.SLACK_WEBHOOK_URL }}
+        if: ${{ env.SLACK_WEBHOOK_URL != '' }}
```

Comment:
Shouldn't this expression be part of the `if` expression on line `120`?

Reply:
true, fixed